### PR TITLE
Disables resin wall weeds

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

Disables weeds crawling up resin walls after being planted.

Also makes your clicks with weapons, ctrl clicks and xeno attacks on the weed transfer directly to the window frame it's on so you don't have to pixel hunt.

## Why It's Good For The Game

The weeds cause issues with visibility.

Also pixel hunting is not fun.

image
image
## Changelog
:cl:
del: Disabled wall weeds
/:cl:

### CODE REVIEW

**Review by Kuro020 — APPROVED** [review]

**Review by RipGrayson — APPROVED** [review]
Approved for whatever it's worth coming from the mapping guy

**Comment by Strayed-Raven** [comment]
finally

File: code/controllers/subsystem/weeds.dm [hunk 1]
```diff
@@ -78,8 +78,6 @@ SUBSYSTEM_DEF(weeds)
 		return
 	var/obj/alien/weeds/weed_to_spawn = node.weed_type
 	var/swapped = FALSE
-	if(iswallturf(T))
-		weed_to_spawn = /obj/alien/weeds/weedwall
```
Comment (Lumipharon) [inline code/controllers/subsystem/weeds.dm:82]: Better to not remove this, since the functionality exists you're just turning it off for walls.

**Comment by Losenis** [comment]
Based, vines on window frames just makes them unclimbable unless you land the funny pixelhunt or alt-click the tile first

**Comment by XElectricX** [comment]
Wall weeds add to the aesthetic. Just disallow them growing on windows if you hate it so much.

File: code/controllers/subsystem/weeds.dm (hunk 1 above)
Comment (ivanmixo) [reply code/controllers/subsystem/weeds.dm:82]: Pointless check, why do we care if it's a wall if walls are unweedable?

**Comment by ClosetedSkeletonGH** [comment]
> Wall weeds add to the aesthetic. Just disallow them growing on windows if you hate it so much.

no they dont, they make resin walls borderline unseeable and normal walls depending on the color of the wall and weed. they provide more issues with how often they glitch out than look good. Like really, theyre more often noticeable as blocking in xenos and causing them to die (since walls dont block xeno vision, not being able to notice a wall is there is a huge issue and I see good players running into walls 24/7).

**Comment by Lumipharon** [comment]
I'd rather just do resin walls for now, since they're far more likely to be more heavily obscured.

**Comment by ivanmixo** [comment]
> I'd rather just do resin walls for now, since they're far more likely to be more heavily obscured, while still having resin under them. While resin again normal walls looks ugo without the wall resin due to the resin edges.

@Kuro020 what's the call?

**Comment by ivanmixo** [comment]
It does not, **but** I've made it so clicking the weed transfers the click onto the widow frame.

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**